thirdparty/github.go

```diff
@@ -83,7 +86,7 @@ func githubShouldRetryWith404s(attempt rehttp.Attempt) bool {
 }
 
 func getGithubClient(token string) (*http.Client, error) {
-	all := rehttp.RetryAll(rehttp.RetryMaxRetries(NumGithubRetries-1), rehttp.RetryTemporaryErr(), githubShouldRetry)
```

Comment:
If the attempt's error was nil we would never retry, since `RetryTemporaryErr()` returns false if attempt.Error is nil.